homeassistant/components/plex/config_flow.py

```diff
@@ -60,13 +60,17 @@ def __init__(self):
 
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user."""
+        errors = {}
         if user_input is not None:
             if user_input.pop("manual_setup", False):
                 return await self.async_step_manual_setup(user_input)
             if CONF_TOKEN in user_input:
                 return await self.async_step_server_validate(user_input)
+            errors["base"] = "no_token"
```

Comment:
Use the `CONF_TOKEN` key when specifying the error key. This will show the error message next to the correct form item.